client/components/signal-warning/index.js

```diff
@@ -8,14 +8,21 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import EditorNotice from 'components/editor-notice';
-import WarningIcon from 'components/icon/warning-circle';
 
 const SignalWarning = () => {
```

Comment:
Nit: maybe we should call it `SignalNotice`/`SignalCountNotice`/`SignalWarningNotice` whichever you prefer... to keep it consistent with other `Notice` components?

We could also potentially put this and `retry-notice` in subdirectories inside `components/notice` since they're basically just specializations of that component. But maybe let's hold off on that for the repo cleanup.

Reply:
Yeah, crossed my mind as well. These are all notices and should be better named/arranged :)